python/src/HSICEstimatorConditionalSensitivity_doc.i.in

```diff
@@ -0,0 +1,33 @@
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::HSICEstimatorConditionalSensitivity
+"Implement a HSIC estimator for conditional analysis.
+
+Available constructors:
+    HSICEstimatorConditionalSensitivity()
+
+    HSICEstimatorConditionalSensitivity(covarianceList, X, Y, estimatorType, weightFunction)
```

Comment:
@KieranDelamotte If you want to have an "available constructors" section (which is not mandatory since there is only one actual constructor), you need to put the names of the arguments in italics (see for instance [AbsoluteExponential](https://openturns.github.io/openturns/master/user_manual/_generated/openturns.AbsoluteExponential.html)).